dotnet/src/Connectors/Connectors.Memory.Qdrant/QdrantGenericDataModelMapper.cs

```diff
@@ -11,7 +11,7 @@
 namespace Microsoft.SemanticKernel.Connectors.Qdrant;
 
 /// <summary>
-/// A mapper that maps between the generic semantic kernel data model and the model that the data is stored in in Qdrant.
+/// A mapper that maps between the generic Semantic Kernel data model and the model that the data is stored in Qdrant.
```

Comment:
The double in is not a typo, but could be better worded as:
`the model that the data is stored under, within Qdrant`